### PR TITLE
add support for invisible recaptcha #120

### LINKED ISSUE #120: Support for Invisible Recaptcha
Any plans to support the Invisible Recaptcha (currently in Beta)?
https://www.google.com/recaptcha/intro/comingsoon/invisiblebeta.html

### PR DESCRIPTION
**Pull request by DylannCordel**
fix #120

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -176,6 +176,24 @@ settings, or by passing ``use_ssl=False`` to the constructor of
 ``ReCaptchaField``.
 
 
+About Invisible ReCAPTCHA
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Currently (207-04-12), Invisible reCAPTCHA is really invisble (no action required for the user)
```
Comment (adamchainz) [inline README.rst]: 2017 not 207

Comment (DylannCordel) [inline README.rst]: oups :stuck_out_tongue: 
Fixed. Thanks !

**Comment by eronaeon** [comment]
Awesome! when will this be merged?

**Comment by adrienbrunet** [comment]
Hello here, what is the status on this PR ? Does it need reviews ? Tests ? Another lib has emerged and this one won't be maintained ? How can I help ? I'd like this feature to be added.. Cheers !

**Comment by mmoravcik** [comment]
As per @adrienbrunet comment, are there any news about this?

**Comment by hendrikschneider** [comment]
Please merge this PR! Invisible recaptcha is already public for quite a long time and this is the biggest recaptcha django addon, which should support it.

**Comment by defbyte** [comment]
hello, are there any outstanding changes (other than the merge conflict resolutions) that need to be made for this PR for Invisible ReCaptcha to ship?

**Comment by robwise1** [comment]
Bump! This would be really helpful to have.

**Comment by AltusBarry** [comment]
Support was added in `1.5.0 (2019-01-09)`

**Pull request closed without merging**